app/assets/stylesheets/application.css.scss

```diff
@@ -0,0 +1,4 @@
+//= require normalize-rails
```

Comment:
No, these are not comments and they are necessary.
The compass gem http://compass-style.org/ parses lines in `application.css.scss` for anything starting with `//=` and then includes them in the build.